server/etc/rc.d/init.d/pulp_workers

```diff
@@ -44,6 +44,7 @@ DEFAULT_LOG_LEVEL="INFO"
 DEFAULT_NODES="celery"
 DEFAULT_CELERYD="-m celery worker --detach"
 DEFAULT_PULP_CONCURRENCY=$(nproc --all)
+DEFAULT_PULP_MAX_TASKS_PER_CHILD=0
```

Comment:
The PULP_MAX_TASKS_PER_CHILD already defines the default of 0 so this probably can be removed.and the logic below reworked to read from the PULP_MAX_TASKS_PER_CHILD itself.